### PR TITLE
Wikidata values added

### LINKED ISSUE #811: Improve Display of Wikidata Identifier on Edition Page
Currently: https://openlibrary.org/books/OL2388827M/The_tale_of_the_Heike
image

- Capitalize 'W' in "wikidata"
- Link QID to `https://www.wikidata.org/wiki/{qid}`

### PR DESCRIPTION
**Pull request by salman-bhai**
Closes #811 

- [x] Wikidata made capital
- [x] href added

P.S. I haven't been able to test if these changes are working locally since I don't have books that have ID Numbers Wikidata on the data that I have in my local environment!

### CODE REVIEW

File: openlibrary/templates/type/edition/view.html [hunk 1]
```diff
@@ -74,8 +74,11 @@
             $return
         <tr>
             <td class="title"><span class="title">$label</span></td>
-            <td class="object"><span class="object" $:cond(itemprop, 'itemprop="%s"' % itemprop, '')>
+            <td class="object"><span class="object" $:cond(itemprop, 'itemprop="%s"' % itemprop, '')>
```
Comment (mekarpeles) [inline openlibrary/templates/type/edition/view.html]: rm trailing whitespace

**Comment by mekarpeles** [comment]
other than extra whitespace, looks good to me

**Comment by salman-bhai** [comment]
@cdrini can you review?

**Comment by mekarpeles** [comment]
I've put this branch on dev.openlibrary.org for testing

**Comment by salman-bhai** [comment]
https://openlibrary.org/books/OL2388827M/The_tale_of_the_Heike

**Comment by tfmorris** [comment]
Looks like it needs a little work: 
https://dev.openlibrary.org/books/OL2388827M/The_tale_of_the_Heike

renders with this literal string:

> Wikidata | Q15977202 = "https://www.wikidata.org/wiki/"+str(id.value) Q15977202

**Comment by tfmorris** [comment]
More generally, I think having the source contain identifier specific code is the wrong approach. The URL template should come from the config data. I'm not sure why Goodreads is special cased, but I think we should figure out if it's really needed and remove it if it's not.

Separately, the class="title" needs to be removed from the labels. It's messing up the rendering of ISBN and LC.

**Comment by salman-bhai** [comment]
@tfmorris can you check it now?

**Comment by salman-bhai** [comment]
> More generally, I think having the source contain identifier specific code is the wrong approach. The URL template should come from the config data. I'm not sure why Goodreads is special cased, but I think we should figure out if it's really needed and remove it if it's not.

Do you think we should do it in this PR or have a separate PR for that?

> Separately, the class="title" needs to be removed from the labels. It's messing up the rendering of ISBN and LC.

Removed and that particular element shouldn't have any class?

**Comment by tfmorris** [comment]
I think adding Wikidata here: https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/upstream/models.py#L112
and to the appropriate config data is probably a better approach.

I don't remember off the top of my head where the config data (e.g. URL templates) is stored. Perhaps @mekarpeles or @hornc know. If not I'm sure I can find it again with a little digging.

**Comment by tfmorris** [comment]
Found it quicker than I expected. The identifier URL templates come from here:

https://openlibrary.org/config/edition.json

but that page needs to be updated by someone with more privileges than I.

**Comment by salman-bhai** [comment]
I don't think I can edit that file as well :p @tfmorris 
Should I close this PR or should we merge this ? @mekarpeles

**Comment by mekarpeles** [comment]
I need a bit more time to review this. If there's a way to merge it it, I'd prefer that. At latest this weekend I'll pull this on dev and do a round of tests + see about editing config/edition.json

**Comment by salman-bhai** [comment]
No issues @mekarpeles !

**Comment by cdrini** [comment]
I think it would be best to follow the convention that's already set (i.e. everything is in https://openlibrary.org/config/edition.json ). I like the changes @tfmorris has suggested to make the process of editing that file more accessible from the codebase, but until then, I think we should just edit that file. Having Wikidata be the *only* thing that doesn't follow that convention seems like bad practice, so I think we should close this PR in favor of @tfmorris ' upcoming solution (or if one of the admins wants to edit the edition.json file directly).

**Comment by tfmorris** [comment]
I've added a new PR which includes the Wikidata entry (among other things) here #881. The piece to update the production instances with that data still needs to be done.

**Comment by salman-bhai** [comment]
Closing this PR as #881 resolves #811

**Pull request closed without merging**